lte/gateway/c/core/oai/tasks/amf/amf_app_ue_context.cpp

```diff
@@ -261,7 +267,7 @@ amf_context_t* amf_context_get(const amf_ue_ngap_id_t ue_id) {
     }
     OAILOG_DEBUG(LOG_NAS_AMF, "Stored UE id " AMF_UE_NGAP_ID_FMT " \n", ue_id);
   }
-  return amf_context_p;
+  OAILOG_FUNC_RETURN(LOG_AMF_APP, amf_context_p);
```

Comment:
Module name is not same in this fun(0. 
General comment to have same module name in one function